VespaGAS/Managers/OilMixCalculator.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+let MLInLiter: Float = 1000
+
+typealias Oil = Volume
+typealias Gasoline = Volume
+
+struct OilMixCalculator {
+    static func oilFor(gasoline: Gasoline, oilMix: OilMix) -> Oil {
```

Comment:
I have implemented this logic now into the OilMix object. Changes are already here.